app/views/configuration/_ui_2.html.haml

```diff
@@ -68,7 +68,8 @@
                :middlewaredeployment                 => ["middleware_deployment_show_list",    _('Middleware Deployments')],
                :middlewaredatasource                 => ["middleware_datasource_show_list",    _('Middleware Datasources')],
                :middlewaredomain                     => ["middleware_domain_show_list",        _('Middleware Domains')],
-               :middlewareservergroup                => ["middleware_server_groups_show_list", _('Middleware Server Groups')]}.each do |resource, (feature, view_name)|
+               :middlewareservergroup                => ["middleware_server_groups_show_list", _('Middleware Server Groups')],
+               :middlewarejms                        => ["middleware_messaging_show_list",     _('Middleware Messaging')]}.each do |resource, (feature, view_name)|
```

Comment:
the key should be middlewaremessaging, not middlewarejms, probably missed in renaming?